src/observation/encodeMatcher/index.js

```diff
@@ -30,7 +30,7 @@ const encodeWhereDescription: WhereDescription => Matcher<*> = description => ra
   } else if (compRight.column) {
     right = (rawRecord: Object)[compRight.column]
   } else {
-    throw new Error('Invalid comparisonRight')
+    right = undefined
```

Comment:
i'm ok with allowing undefined, but this is not the correct solution. `compRight.value !== undefined` ought to be modified so that it checks for a property's presence, and not value.

Reply:
Right. I will correct

Reply:
@leninlin Hey, you there? do you want to finish up this PR?

Reply:
Oh, sorry. I forgot. I will finish this.